fboss/platform/configs/tahan800bc/platform_manager.json

```diff
@@ -12,6 +12,15 @@
       },
       "pmUnitName": "TAHAN"
     },
+    "SMB_FRU_SLOT": {
```

Comment:
Since "SMB_SLOT" is renamed as "TAHAN_SLOT", can we rename "SMB_FRU_SLOT" to "SMB_SLOT" please?

Reply:
1.Hi , You have a point.Named to SMB_FRU_SLOT is because in this slot we only put the versioned sensors not all SMB sensors, SMB_SLOT is a big scope, we just avoid the misunderstanding and confusion.That’s why it’s named that.
![image](https://github.com/user-attachments/assets/75748b5a-dbf4-4dfe-bc8c-c4f936d247b2)

2.If you insist on modifying it, I can do so and wait for your reply, thank you!

Reply:
Ok, that's a good point.According to your suggestion i have modified it and tested pass. Committed it again.Please review, thanks.

![image](https://github.com/user-attachments/assets/69eb16aa-8725-4e00-819b-7cf367b86bb8)
![image](https://github.com/user-attachments/assets/74f09f9d-2b05-44dd-a037-a557b5dd2d86)
<img width="884" alt="image" src="https://github.com/user-attachments/assets/af34272f-19a0-4ba3-8d6f-0909ebebfc03">